quickfixj-core/src/main/java/quickfix/mina/acceptor/AbstractSocketAcceptor.java

```diff
@@ -227,11 +229,20 @@ private void createSessions(SessionSettings settings) throws ConfigError, FieldC
             }
 
             if (connectionType.equals(SessionFactory.ACCEPTOR_CONNECTION_TYPE)) {
-                AcceptorSocketDescriptor descriptor = getAcceptorSocketDescriptor(settings, sessionID);
-                if (!isTemplate) {
-                    Session session = sessionFactory.create(sessionID, settings);
-                    descriptor.acceptSession(session);
-                    allSessions.put(sessionID, session);
+                try {
+                    AcceptorSocketDescriptor descriptor = getAcceptorSocketDescriptor(settings, sessionID);
+                    if (!isTemplate) {
+                        Session session = sessionFactory.create(sessionID, settings);
+                        descriptor.acceptSession(session);
+                        allSessions.put(sessionID, session);
+                    }
+                } catch (Throwable t) {
```

Comment:
Not a big fan of catching general `Throwable` but made code as similar as possible to AbstractSocketInitiator. Could be changed later.